src/deepqmc/wf/nn_wave_function/env.py

```diff
@@ -28,15 +46,33 @@ def __call__(self, diffs):
             if self.isotropic
             else norm(jnp.einsum('ers,ies->ier', self.zetas, d), safe=True)
         )
-        return jnp.exp(-exponent)
-
-    @classmethod
-    def from_mol(cls, mol, **kwargs):
-        r"""Create the input of the constructor from a :class:`~deepqmc.Molecule`."""
-        shells = []
-        for i, (z, n_shell, n_pp_shell) in enumerate(
-            zip(mol.charges, mol.n_shells, mol.n_pp_shells)
-        ):
-            for k in range(n_pp_shell, n_shell):
-                shells.append((i, z / (k + 1)))
-        return cls(shells, **kwargs)
+        return self.mo_coeff(jnp.exp(-exponent))
+
+
+class FermiNetEnvelopes(hk.Module):
```

Comment:
I was on the fence about making this a separate class or including it as an option in our current envelopes. In the end I think there are sufficiently many differences that having separate classes results in more readable code.

Reply:
I would vote for keeping the two implementations as a single class. As discussed offline the `ExponentialEnvelopes` are a special case of the `FermiNetEnvelopes`, where some of the parameters (the exponential coefficients) are shared across orbitals.

Reply:
Resolved in https://github.com/deepqmc/deepqmc/pull/121/commits/752915f78e9021305b3644f31b985893732bb9f9